pkg/drivers/kic/oci/oci.go

```diff
@@ -257,6 +257,25 @@ func createContainer(ociBin string, image string, opts ...createOpt) error {
 	return nil
 }
 
+// StartContainer starts a container with "docker/podman start"
+func StartContainer(ociBin string, container string) error {
+	// construct the actual docker start argv
+	args := []string{"start"}
+
+	// to run nested container from privileged container in podman https://bugzilla.redhat.com/show_bug.cgi?id=1687713
+	if ociBin == Podman {
+		args = append(args, "--cgroup-manager", "cgroupfs")
```

Comment:
I think this flag is not available on podman-cli on mac.
u might wanna dd if ciBin == Podman && goos == "linux"

Reply:
Done (it wasn't working for "run" either, though)